### PR TITLE
implemented BookController, dto, mapstruct and mapper

### PR DESCRIPTION
**Pull request by irynamekh**

### CODE REVIEW

**Review by OleksandrSaven — APPROVED** [review]
LGTM!

File: src/main/java/com/bookstore/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "books")
```
Comment (vovasalyha) [inline src/main/java/com/bookstore/controller/BookController.java:17]:
```suggestion
@RequestMapping("/api/books")
```

File: src/main/java/com/bookstore/dto/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```
Comment (vovasalyha) [inline src/main/java/com/bookstore/dto/BookDto.java:7]: It's OK to return ID in response.

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**